genie-agent/src/main/java/com/netflix/genie/agent/execution/statemachine/actions/LaunchJobAction.java

```diff
@@ -32,7 +32,6 @@
 import lombok.extern.slf4j.Slf4j;
 
 import java.io.File;
-import java.util.Map;
 
 /**
  * Action performed when in state LAUNCH_JOB.
```

Comment:
Somewhere, now that you're relying on bash, don't we need to assert that bash is available or short circuit the agent?

Reply:
It's non trivial... we use `env` in the shebang to ask the shell to run the script with the most appropriate `bash` available. It could be at a non-default location, be named something else other than `bash` and a million other things.

In general, seems to me like trying figure out whether bash is installed is not worth it. 

Moreover, if bash was indeed not installed, a user would get a message in `stdout` (terminal or file depending on `--interactive`.